qualification/antenna_channelised_voltage/test_channel.py

```diff
@@ -158,3 +159,37 @@ async def test_channelisation_and_sfdr(
             verticalalignment="top",
         )
     pdf_report.figure(fig)
+
+    zoom_plot_range = 3
+
+    assert plot_channel >= zoom_plot_range and plot_channel <= receiver.n_chans - zoom_plot_range - 1
+    assert zoom_plot_range >= 3
+
+    if (plot_channel - zoom_plot_range) < 0:
+        zoom_plot_range = np.abs(plot_channel - zoom_plot_range)
+
+    if (plot_channel + zoom_plot_range) > receiver.n_chans:
+        zoom_plot_range = receiver.n_chans - plot_channel
```

Comment:
These checks won't actually do anything, I think, because of the asserts that you've got above.